aws/configs/amazon-cloudwatch-agent.json

```diff
@@ -1,29 +1,12 @@
 {
     "agent": {
         "logfile": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
-        "metrics_collection_interval": 10,
+        "metrics_collection_interval": 60,
         "run_as_user": "root"
     },
     "metrics": {
-        "namespace": "DefaultWorkerPool",
+        "namespace": "SpaceliftWorkerImage",
```

Comment:
How about `SpaceliftWorker`? Just on the grounds that although this repo is building the image, the metrics are actually produced by instances of that image.

Reply:
good point, done